conda/conda-releaselogs/azure-ai-vision.md

```diff
@@ -0,0 +1,7 @@
+Azure AI Vision client library for Python (conda)
```

Comment:
Should this be Azure AI vision imageanalysis? To not confuse with the (deprecated) azure-ai-vision library...

Reply:
Do you know if we will have more vision libraries? If yes, we want to bundle them together.